docs/Helm.md

```diff
@@ -0,0 +1,106 @@
+# Splunk Operator Helm Installation
+
+## Splunk Operator Helm chart Repository
+
+Add the Splunk Operator and Enterprise charts to your Helm repository.
+
+```
+helm repo add splunk https://splunk.github.io/splunk-operator/
+helm repo update
+```
+
+The ```splunk``` chart repository contains the ```splunk/splunk-operator``` chart to deploy the Operator and the ```splunk/splunk-enterprise``` chart to deploy Splunk Custom Resources.
+
+Helm provides a long list of commands to manage your deployment, we'll be going over a few useful ones in the sections to come. You can learn more about supported commands [here](https://helm.sh/docs/helm/helm/).
+
+## Splunk Operator deployments
+
+To install the Splunk Operator:
+
+```
+helm install <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+To upgrade the operator, specify override configurations through the CLI or a new file:
+
+```
+helm upgrade --set splunkOperator.clusterWideAccess=false <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+Here we've upgraded the operator by revoking cluster-wide access.
+
+To rollback the operator to a previous release:
+
+```
+helm rollback <RELEASE_NAME> <REVISION_NUMBER> -n <RELEASE_NAMESPACE>
+```
+
+To clean-up resources and uninstall the operator:
+
+```
+helm uninstall <RELEASE_NAME> -n <RELEASE_NAMESPACE>
+```
+
+### Configuring Splunk Operator deployments
+
+There are a couple ways you can configure your operator deployment
+
+1. Using a ```new_values.yaml``` file to override default values (Recommended)
+```
+helm install -f new_values.yaml <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+2. Using the Helm CLI to set new values
+```
+helm install --set <KEY>=<VALUE> <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+Read more about configuring values [here](https://helm.sh/docs/intro/using_helm/).
+
+## Splunk Enterprise deployments
+
+The Splunk Enterprise chart allows you to install and configure Splunk Enterprise custom resources.
+
+First build the Splunk Operator chart as a dependency:
+```
+helm dependency build splunk/splunk-enterprise
+```
+
+To install a configured Splunk Enterprise deployment:
+```
+helm install <RELEASE_NAME> splunk/splunk-enterprise -n <RELEASE_NAMESPACE>
```

Comment:
nit: Any significance for the word RELEASE in RELEASE_NAMESPACE?

Reply:
When using the any of the commands ```install```, ```upgrade```, ```rollback```, ```uninstall``` for a specific release, the release's namespace must be specified. Including it here is just to be explicit to which namespace we're referring to.